src/components/common/ChainIndicator/index.tsx

```diff
@@ -1,3 +1,5 @@
+import { getChainLogo } from '@/config/chains'
+import Image from 'next/image'
```

Comment:
I thought we weren't using this component. Does it add some size optimizations with SSG?

Reply:
Nice.

Reply:
Looks like its causing our jest tests to fail though so I will change it to `<img>`